modules/serverless-create-domain-mapping-kn.adoc

```diff
@@ -0,0 +1,60 @@
+[id="serverless-create-domain-mapping-kn_{context}"]
+= Creating a custom domain mapping using the Knative CLI
+
+You can use the `kn` CLI to create a `DomainMapping` resource that maps to an Addressable target reference, such as a Knative service or a Knative route.
+
+The `--ref` flag specifies an Addressable target reference for domain mapping.
+
+If a prefix is not provided when using the `--ref` flag, it is assumed that the target is a Knative service in the current namespace. The examples in the following procedure show the prefixes for mapping to a Knative service or a Knative route.
```

Comment:
To be more clear, should you say _If a namespace is not provided..._. I was a bit confused by _prefix_ as the `service_namespace` comes after `service_name`.